account_invoice_inter_company/tests/test_inter_company_invoice.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+# © 2015-2017 Chafique Delli <chafique.delli@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase
+from odoo import _
+
+
+class TestAccountInvoiceInterCompany(TransactionCase):
+    def setUp(self):
+        super(TestAccountInvoiceInterCompany, self).setUp()
+        self.wizard_obj = self.env['wizard.multi.charts.accounts']
+        self.account_obj = self.env['account.account']
+        self.invoice_obj = self.env['account.invoice']
+        self.invoice_company_a = self.env.ref(
+            'account_invoice_inter_company.customer_invoice_company_a')
+        self.user_company_a = self.env.ref(
+            'account_invoice_inter_company.user_company_a')
+        self.user_company_b = self.env.ref(
+            'account_invoice_inter_company.user_company_b')
+
+        # Install COA for company A and B
+        wizard_comp_a = self.wizard_obj.create({
+            'company_id': self.env.ref(
+                'account_invoice_inter_company.company_a').id,
+            'chart_template_id': '1',
```

Comment:
Shouldn't `chart_template_id` be an integer here? I see string `'1'` instead.
Strange that the test pass, anyway could you please fix it?
Same discourse on line 39.

Reply:
@astirpe , I fixed the value for chart_template_id .